### DISCUSSION TITLE
Enclosure attached to power socket outlet - 3D-printed

### DISCUSSION THREAD

**Discussion by stritti** [tbnobody/OpenDTU#2084] [Show and tell]
Hi,
I wanted to attach OpenDTU directly to a socket without cable and created following 3d printable model. Probably someone else find it useful:

https://www.printables.com/model/919028-opendtu-socket-outlet-enclosure

![image](https://github.com/tbnobody/OpenDTU/assets/184547/6ebcd95f-42db-4682-8756-39b4259209cb)

Looking forward your feedback and improvements.

Probably someone has an idea how to replace the USB power supply to reduce depth of the case?

Regards Stephen

**Comment by ghost** [comment]
Hi.
also es gibt Schalterdosen- / Unterputz-Netzteile. Meist so um die 25 - 50 €.

Alternativ aus Ch: 
[Mini AC-DC Netzteil](https://de.aliexpress.com/item/1005003691590074.html?)
Sind kleiner, allerdings meist mit kaum mehr wie 600mA, 2A muss man schon sehr suchen. Gedacht für Leiterplattenmontage, es gibt die auch bedrahtet. Und gibt es auch ohne Gehäuse.

**Reply by stritti** [reply]
Hi, danke für den Tipp. Mal sehen ob ich das noch umbaue. Am liebsten wäre mir etwas ohne noch zusätzlich löten.

**Reply by ghost** [reply]
Das steckt an.
Interessehalber mal nach ["Steckernetzteil flach"](https://www.google.com/search?q=Steckernetzteil+flach&sca_esv=cfcd3706826e13a2&sca_upv=1&sxsrf=ADLYWIIhUUxAUIGz-z9yi3uI2Jlj8-W2-g%3A1719263212898&ei=7N95ZrbANuSdi-gP6cepmAc&ved=0ahUKEwi25tHIkvWGAxXkzgIHHeljCnMQ4dUDCBA&oq=Steckernetzteil+flach&gs_lp=Egxnd3Mtd2l6LXNlcnAiFVN0ZWNrZXJuZXR6dGVpbCBmbGFjaEgAUABYAHAAeAGQAQCYAQCgAQCqAQC4AQzIAQCYAgCgAgCYAwCSBwCgBwA&sclient=gws-wiz-serp) gesucht.
Da kommen ganz interessante Ergebnisse. Netzteile in Größe von etwa eines ganzen Schuko Steckers. oder ähnlich.
Vielleicht ist etwas für dich dabei.
[Brennenstuhl USB-Ladegerät 1508110, 12W, 2,4A, weiß, 1x USB-A, 1 Port](https://www.bueromarkt-ag.de/usb-ladegeraet_brennenstuhl_1508110_12w_2_4a,p-1508110,l-google-css,pd-b2c.html?gad_source=1&gclid=Cj0KCQjwsuSzBhCLARIsAIcdLm6l0xgUJIbx7AXYNv6NSORMPPEo0ysDkXcZtxie42-b-KIFPzJS6sIaAqI3EALw_wcB)
[Legrand USB Adapter 050681](https://www.elektronetshop.de/product/legrand-usb-adapter-050681.204541.html?p=16&gad_source=1&gclid=Cj0KCQjwsuSzBhCLARIsAIcdLm60gtL6i1r0n1rNrsSE_UxQAmXlaysjrP8D2OCcGAgKgr8kYPX-T5QaAgO3EALw_wcB)
[LogiLink USB-Ladegerät 10.5W Innenbereich, Steckdose Ausgangsstrom (max.) 2100mA Anzahl Ausgänge: 1 x USB-A](https://www.voelkner.de/products/5889236/LogiLink-USB-Ladegeraet-10.5W-Innenbereich-Steckdose-Ausgangsstrom-max.-2100mA-Anzahl-Ausgaenge-1-x-USB-A.html?ref=43&offer=0&utm_source=google&utm_medium=organic&utm_campaign=fpla&campaign_type=pla&pla_campaign_no=14526503487&campaign_content=143304995674&gad_source=1&gclid=Cj0KCQjwsuSzBhCLARIsAIcdLm4SGaZI1-zqCGhMvT8TO9-eXRuZ-NAmWVVrVHNOqz0SrJL9Z4l9iPoaAh7EEALw_wcB)
usw.

**Reply by stritti** [reply]
Oh cool, ja genau sowas ist perfekt! Vielen Dank, die hatte ich nicht entdeckt. Da muss ich wohl mal was probieren 😇
Vielen lieben Dank!
